autopush/endpoint.py

```diff
@@ -254,6 +273,49 @@ def _db_error_handling(self, d):
         d.addErrback(self._response_err)
         return d
 
+    def _extract_jwt(self, token, crypto_key):
+        # first split and convert the jwt.
+        sig = None
+        payload = {}
+
+        def fix_padding(string):
+            return string + '===='[len(sig) % 4:]
```

Comment:
I assume this is for right padding? If so....
```python
sig = "{:=<4}".format(sig)
```
per https://docs.python.org/2/library/string.html#formatexamples under 'Aligning the text and specifying a width'.

As is, its quite confusing that this inline function uses sig from outside its scope, even though the string its passed is the sig on line 286, but then on line 292 it right-pads the crypto_key based on the sig length?